packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -214,6 +214,11 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
       focusZoneProps,
       disabled = false,
     } = this.props;
+
+    let totalNestingDepth = this.props.totalNestingDepth;
+    if (totalNestingDepth === undefined) {
+      totalNestingDepth = groupNestingDepth;
```

Comment:
fallback for when used inside by onRenderCell of GroupedList.  A separate totalNestingDepth is not required there as no need to align with any headers.